README.md

```diff
@@ -22,6 +22,35 @@ cls.fit(X_train, y_train)
 predictions = cls.predict(X_test)
 ```
 
+## auto-sklearn acceleration with sklearnex
+
+You can accelerate auto-sklearn with [sklearnex](https://github.com/intel/scikit-learn-intelex). It is extension that speedups sklearn algorithms through patching with highly optimized machine learning kernels.
```

Comment:
multiple suggestions with the main mistakes corrected:

```suggestion
You can accelerate auto-sklearn with [sklearnex](https://github.com/intel/scikit-learn-intelex). It is an extension that speeds up scikit-learn algorithms through patching with highly optimized machine learning kernels.
```

it's also a good idea to use the full name of the extension:

```suggestion
You can accelerate auto-sklearn with [Intel(R) Extension for Scikit-Learn (sklearnex)](https://github.com/intel/scikit-learn-intelex). It is an extension that speeds up scikit-learn algorithms through patching with highly optimized machine learning kernels.
```

I have doubts about the `through patching with highly optimized machine learning kernels` phrase, I don't think it's clear enough for the reader, so here is a way to rephrase this:

```suggestion
You can accelerate auto-sklearn with [Intel(R) Extension for Scikit-Learn (sklearnex)](https://github.com/intel/scikit-learn-intelex). The acceleration is achieved through **patching**: replacing scikit-learn algorithms with their optimized versions provided by the extension.
```

you may also refer users to our docs where patching is explained: https://intel.github.io/scikit-learn-intelex/what-is-patching.html